configure.ac

```diff
@@ -613,7 +633,7 @@ if test "X$HAVE_SZIP" = "Xyes" -a "x$HAVE_SZLIB_H" = "xyes"; then
 
     AC_SUBST([LL_PATH]) LL_PATH="$LD_LIBRARY_PATH"
 
-    AC_TRY_RUN([
+    AC_RUN_IFELSE([AC_LANG_SOURCE([[
```

Comment:
The macro changes in the rest of the file were upgraded by autoupdate to be nicer for cross-compiling